pages/profiles/lucy811.md

```diff
@@ -0,0 +1,36 @@
+
+Hello, everyone! My name is Lucy. I am a recent graduate student in _Illinois Institute of Technology_. My major is **computer science**. Nice to be your team.
+
+* * * *
+
+First Header|Second Header
```

Comment:
* I think you can give a title to the table here instead of the "First header" and "second header"